plugins/destination/bigquery/go.mod

```diff
@@ -3,19 +3,24 @@ module github.com/cloudquery/cloudquery/plugins/destination/bigquery
 go 1.19
 
 require (
-	cloud.google.com/go/bigquery v1.49.0
-	github.com/cloudquery/plugin-sdk v1.45.0
+	cloud.google.com/go/bigquery v1.50.0
+	github.com/apache/arrow/go/v13 v13.0.0-20230509040948-de6c3cd2b604
+	github.com/cloudquery/plugin-pb-go v1.0.8
+	github.com/cloudquery/plugin-sdk/v3 v3.5.1
+	github.com/goccy/go-json v0.9.11
 	github.com/rs/zerolog v1.29.0
 	golang.org/x/sync v0.1.0
 	google.golang.org/api v0.114.0
 )
 
+replace github.com/cloudquery/plugin-sdk/v3 v3.2.0 => ../../../../plugin-sdk
```

Comment:
is it still required?

Reply:
Sorry this wasn't actually ready for review, I thought it was still in draft